tests/integration_tests/test_grpc_only.py

```diff
@@ -58,33 +60,43 @@ def test_grpc_mode(custom_ethermint):
         "to": contract.address,
         "data": contract.encodeABI(fn_name="currentChainID"),
     }
-    api_port = ports.api_port(custom_ethermint.base_port(2))
+    api_port = ports.api_port(custom_ethermint.base_port(1))
     # in normal mode, grpc query works even if we don't pass chain_id explicitly
-    rsp = grpc_eth_call(api_port, msg)
-    print(rsp)
-    assert "code" not in rsp, str(rsp)
-    assert 9000 == int.from_bytes(base64.b64decode(rsp["ret"].encode()), "big")
-
+    success = False
+    for i in range(3):
+        rsp = grpc_eth_call(api_port, msg)
+        print(i, rsp)
```

Comment:
I assume this is not necessary

Reply:
[print](https://github.com/evmos/ethermint/blob/7f196ce451131ab5346d1635a48e435c69f592ba/tests/integration_tests/test_grpc_only.py#L64) was here before, no harms to print out the retry?

Reply:
let's remove it since it's not useful for the test and it's just creating more logs